### DISCUSSION TITLE
JQuery - Problema ao fazer requisição a uma página em localhost

### DISCUSSION THREAD

**Discussion by danilocsf** [frontendbr/forum#1414] [Perguntas e Respostas]
Boa tarde pessoal!

Trabalho atualmente em uma aplicação que deve funcionar offline.
Há uma biblioteca que faz o roteamento das páginas. 
Para tanto, em determinado momento 'executado o seguinte código:
 $.get(page).done(function (html)  ....

Esse page não é o caminho completo, apenas o caminho da página:
Por exemplo: /custom_field_tree/custom_field_tree.template.html 

Porém não está funcionando offline e, pelo que pesquisei deveria funcionar offline.
Dá o seguinte erro:
 http://localhost:8090/custom_field_tree/custom_field_tree.template.html net::ERR_INTERNET_DISCONNECTED

Alguém saberia me ajudar com esse problema por favor?

**Comment by kivervinicius** [comment]
Pesquise sobre ServiceWorkers ou AppCache, seria a melhor solução para você, se está solicitando via http offline não vai funcionar mesmo.
